metadata-utils/src/main/java/com/linkedin/metadata/authorization/PoliciesConfig.java

```diff
@@ -93,6 +93,11 @@ public class PoliciesConfig {
       "Create Domains",
       "Create new Domains.");
 
+  public static final Privilege CREATE_POSTS_PRIVILEGE = Privilege.of(
+      "CREATE_POSTS",
```

Comment:
Not just create POSTS right? This should be domain specific -> "Create Global Announcement" or something similar.

In the future, if end up supporting Entity Page posts, that will be permissioned as normal with consideration for the entity urn

Reply:
Good point, this has been changed!